app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -430,6 +433,28 @@ class TabsTrayFragment : AppCompatDialogFragment() {
             }.show()
     }
 
+    @VisibleForTesting
+    internal fun showBookmarkSnackbar(
+        tabSize: Int
+    ) {
+        val anchor = if (requireComponents.settings.accessibilityServicesEnabled) {
+            null
+        } else {
+            new_tab_button
+        }
```

Comment:
Could we put this is a common getter for `showCollectionSnackbar` to also share?

Reply:
Good idea.  I'll refactor.